### PR TITLE
Comments popup

### PR DESCRIPTION
**Pull request by NoelLincoln**
Hello 👋, @Surafels  @FilbertKimasa 

I have implemented the comments popup as per the requirements:

- Display selected Movie's comments and details
- Design page as per wireframe

### CODE REVIEW

File: src/modules/commentsPopup.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
+/* eslint-disable no-underscore-dangle */
+const openCommentWindow = (movie) => {
+  const CommentsPopup = document.querySelector('.comments-popup');
+  CommentsPopup.style.display = 'flex';
+
+  CommentsPopup.innerHTML = `<div class="top">
+                // eslint-disable-next-line no-underscore-dangle
```
Comment (FilbertKimasa) [inline src/modules/commentsPopup.js:7]: Please consider removing that commented line, it displays on the popup. Look at the attached image below
noel2

File: src/modules/commentsPopup.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+                <img class="movie-image" src="${movie._embedded.show.image.medium}"></img>
```
Comment (FilbertKimasa) [inline src/modules/commentsPopup.js:8]: The popup image is not properly styled. could you please fix that? Look at the image below.
noel image

**Review by FilbertKimasa — COMMENTED** [review]
Hello @NoelLincoln  👋, 

Great work and effort you put into this project so far.

## Highlights 🚀

- [ ] Popup window display on comment button click
- [ ] The popup window displays the data fetched from the API
- [ ] Descriptive PR
- [ ] Correct liters

> I'd like to remind you that your long hours of work are appreciated. However, the code review process is for improving your code, not criticizing it. 😄

 ## Required Changes ♻️
_Please check the comments under the review._

Cheers, and Happy coding!👏👨‍💻💻

Feel free to mention @FilbertKimasa  to leave any questions or comments in the PR thread if something is not 100% clear.

File: src/modules/commentsPopup.js (hunk 2 above)
Comment (FilbertKimasa) [inline src/modules/commentsPopup.js:7]: The popup image is not properly styled. could you please fix that? Look at the image below.
noel image

File: src/modules/commentsPopup.js (hunk 1 above)
Comment (FilbertKimasa) [inline src/modules/commentsPopup.js]: Please consider removing that commented line, it displays on the popup. Look at the attached image below
noel2

**Review by FilbertKimasa — CHANGES_REQUESTED** [review]
Hello @NoelLincoln  👋, 

Great work and effort you put into this project so far.

## Highlights 🚀

- [ ] Popup window display on comment button click
- [ ] The popup window displays the data fetched from the API
- [ ] Descriptive PR
- [ ] Correct liters

> I'd like to remind you that your long hours of work are appreciated. However, the code review process is for improving your code, not criticizing it. 😄

 ## Required Changes ♻️
_Please check the comments under the review._

Cheers, and Happy coding!👏👨‍💻💻

Feel free to mention @FilbertKimasa  to leave any questions or comments in the PR thread if something is not 100% clear.

File: src/modules/commentsPopup.js (hunk 1 above)
Comment (NoelLincoln) [reply src/modules/commentsPopup.js]: Oh, I did not see that. Thanks. I have corrected it

File: src/style.css [hunk 3]
```diff
@@ -109,3 +109,63 @@ footer {
...
+  position: fixed;
+  flex-direction: column;
+  gap: 20px;
+  align-items: center;
+  border: 1px solid;
+  z-index: 11;
+  box-sizing: border-box;
+  width: 90%;
+  margin: auto;
+  height: 90vh;
```
Comment (Surafels) [inline src/style.css:124]: - [ ] Optional  suggestions

Kindly recommend you increase the height of your popup window

**Review by Surafels — APPROVED** [review]
Hello @NoelLincoln 
Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

- [ ] Highlights
-✔️ No linter Errors
✔️ Fetch movies from API
✔️ Required changes implemented according to the preview reviewer

File: src/modules/commentsPopup.js (hunk 1 above)
Comment (FilbertKimasa) [reply src/modules/commentsPopup.js]: 👍

**Pull request merged**